Makefile

```diff
@@ -88,6 +88,12 @@ LOG=ANSIBLE_LOG_PATH="$(LOGFILE)"
 LOGINFO=$(INFO) "Wrote $(LOGFILE)"
 endif
 
+ifndef ANSIBLE_COLLECTIONS_PATH
+LINT_ANSIBLE_COLLECTIONS_PATH=${PWD}/collections:${HOME}/.ansible/collections:/usr/share/ansible/collections
```

Comment:
would like to have done an ansible --version|grep 'ansible collections location' to obtain the default path(s), but that would require having the venv environment setup before hand.